redmine commands - handle multiselect customfield (#34643)
demisto/content: Packs/Redmine/ReleaseNotes/1_0_4.md

#### Integrations

##### Redmine

> - Updated the custom_fields argument in the **redmine-issue-create**, **redmine-issue-update** to handle multiselect fields.

Reviewer: ```suggestion
- Fixed an issue where custom_fields argument in the **redmine-issue-create**, **redmine-issue-update** didn't handle multiselect fields.
```

> - Fixed an issue where custom_fields argument in the **redmine-issue-create**, **redmine-issue-update** didn't handle multiselect fields.

Reviewer 2: ```suggestion
Fixed an issue where the *custom_fields* argument in the ***redmine-issue-create*** and ***redmine-issue-update*** commands didn't handle multiselect fields.
```
